tests/example-syntax-error/expected_results.json

```diff
@@ -0,0 +1,2 @@
+{"version": 2, "status": "error", "tests": [], "message": "/tmp/example-syntax-error/bob.f90:37:47:\\n\\n   37 |   function hey(statement_THIS_IS_A_SYNTAX_ERROR)\\n      |                                               1\\nError: Symbol ‘statement_this_is_a_syntax_error’ at (1) has no IMPLICIT type\\n/tmp/example-syntax-error/bob.f90:39:45:\\n\\n   39 |     character(len=*), intent(in) :: statement\\n      |                                             1\\nError: Symbol at (1) is not a DUMMY variable\\nmake[2]: *** [CMakeFiles/bob.dir/build.make:75: CMakeFiles/bob.dir/bob.f90.o] Error 1\\nmake[1]: *** [CMakeFiles/Makefile2:622: CMakeFiles/bob.dir/all] Error 2\\nmake: *** [Makefile:101: all] Error 2\\n"
```

Comment:
This is hard to read and review manually.

Perhaps you could format this test case and then use a Python or jq script to compare the two json files? That way you avoid tests failing due to slightly different formatting as well, since the structure and content is compared in the test.

Reply:
In case it's helpful, here's the Julia CI flow to compare the results generated by the test-runner with expected JSON files: https://github.com/exercism/julia-test-runner/blob/main/.github/workflows/ci.yml

Reply:
Thanks for the suggestion. I simply followed the c++ as template. I will have look at jq and the Julia implementation sometime later this week

Reply:
@pclausen I've added some commits that use jq for the results.json files.